src/main/java/hudson/plugins/promoted_builds/inheritance/JobPropertyImplSelector.java

```diff
@@ -1,7 +1,8 @@
 
 package hudson.plugins.promoted_builds.inheritance;
 
-import org.apache.log4j.Logger;
```

Comment:
This guy was coming from Acegi Security in the core, but the transient dependency was excluded in 1.625.3: https://github.com/jenkinsci/jenkins/blob/jenkins-1.625.3/core/pom.xml#L404-L407

Probably the extension class was never able to load correctly... But it was optional, so Jenkins just ignored that, yey.